torch/nn/parallel/distributed.py

```diff
@@ -309,10 +309,8 @@ class DistributedDataParallel(Module):
                    only be provided when the input module resides on a single
                    CUDA device. For single-device modules, the i'th
                    :attr:`module` replica is placed on ``device_ids[i]``. For
```

Comment:
Don't think these docs are accurate anymore since `device_ids` has to contain only one element right?

Reply:
Updated. PTAL